### PR TITLE
Collection of Devices

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

**Review by mariusgiger — COMMENTED** [review]
looks good, a few comments

File: types/mclimate/vicki/uplink.js [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+  });
+  return resultToPass;
+}
+
+function decodeUplink(bytes) {
+  let data = {};
+  const resultToPass = {};
+
+  if (bytes[0].toString(16) === 1 || bytes[0].toString(16) === 129) {
+    data = mergeObj(data, handleKeepalive(bytes, data));
```
[the remark this answers is not in the document]
Comment (J2thatsme) [reply types/mclimate/vicki/uplink.js:275]: No there are also "default" values.

[the remark this answers is not in the document]
Comment (J2thatsme) [reply types/mclimate/vicki/uplink.js:275]: Good point. It was an already created decoder i just copied it ^^

File: types/ewattch/squid-hc-lorawan/lifecycle.schema.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "$id": "https://akenza.io/ewattch/squid-hc-lorawan/lifecycle.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Lifecycle",
+  "type": "object",
+  "properties": {
+    "batteryStatus": {
```
[the remark this answers is not in the document]
Comment (J2thatsme) [reply types/ewattch/squid-hc-lorawan/lifecycle.schema.json:7]: Its a class C device, so its hooked up to electricity.

File: types/digitalMatter/oyster3/stats_v3.schema.json [hunk 3]
```diff
@@ -0,0 +1,108 @@
+{
+  "$id": "https://akenza.io/digitalMatter/oyster3/stats_v3.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Stats V3",
```
[the remark this answers is not in the document]
Comment (mariusgiger) [reply types/digitalMatter/oyster3/stats_v3.schema.json:4]: we should use different device types in this case

**Review by mariusgiger — APPROVED** [review]

File: types/digitalMatter/oyster3/stats_v3.schema.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (J2thatsme) [reply types/digitalMatter/oyster3/stats_v3.schema.json:4]: thats why its oyster 3 and not oyster ^^
Its till possible to send the old stats so i included both

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**